src/interp/interp-inl.h

```diff
@@ -351,16 +351,40 @@ bool operator!=(const RefPtr<U>& lhs, const RefPtr<V>& rhs) {
 }
 
 //// ValueType ////
-inline bool IsReference(ValueType type) { return type.IsRef(); }
-template <> inline bool HasType<s32>(ValueType type) { return type == ValueType::I32; }
-template <> inline bool HasType<u32>(ValueType type) { return type == ValueType::I32; }
-template <> inline bool HasType<s64>(ValueType type) { return type == ValueType::I64; }
-template <> inline bool HasType<u64>(ValueType type) { return type == ValueType::I64; }
-template <> inline bool HasType<f32>(ValueType type) { return type == ValueType::F32; }
-template <> inline bool HasType<f64>(ValueType type) { return type == ValueType::F64; }
-template <> inline bool HasType<Ref>(ValueType type) { return IsReference(type); }
-
-template <typename T> void RequireType(ValueType type) {
+inline bool IsReference(ValueType type) {
+  return type.IsRef();
+}
+template <>
+inline bool HasType<s32>(ValueType type) {
+  return type == ValueType::I32;
+}
+template <>
+inline bool HasType<u32>(ValueType type) {
+  return type == ValueType::I32;
+}
+template <>
+inline bool HasType<s64>(ValueType type) {
+  return type == ValueType::I64;
+}
+template <>
+inline bool HasType<u64>(ValueType type) {
+  return type == ValueType::I64;
+}
+template <>
+inline bool HasType<f32>(ValueType type) {
+  return type == ValueType::F32;
+}
+template <>
+inline bool HasType<f64>(ValueType type) {
+  return type == ValueType::F64;
+}
+template <>
+inline bool HasType<Ref>(ValueType type) {
+  return IsReference(type);
+}
```

Comment:
Can we keep the LHS here?  Seems more readable

Reply:
I added
```
AllowShortFunctionsOnASingleLine: true
AlwaysBreakTemplateDeclarations: MultiLine
```
to 1. Allow short functions on an single line 2. Not break after `template<..>` when the whole function declaration can fit within a single line. After that this becomes
```cpp
template <> inline bool HasType<Ref>(ValueType type) {                           
  return IsReference(type);                                                      
} 
```
Even with `AllowShortFunctionsOnASingleLine: true`, we can't still fit this within a single line like LHS because it exceeds 80 cols. I think we can exclude some code blocks from 80 cols limit if that helps readability, but I'm not sure this place warrants that exception. I actually prefer this 80 cols-complying version. What do you think?

If functions are shorter than this so that they fit in 80 cols, they will now be allowed on a isngle line.

Reply:
It seems worth preserving the existing formatting where all the types line up and we have one-line-per-type ..  I think this is just limited to this one file, so it should be possible to make an an exception.

Reply:
Wrap that part with `// clang-format off` and `on`.